src/RouteAttributesRegistrarInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Router;
+
+/**
+ * `RouteAttributesRegistrarInterface` allows registering routes that declared in classes/functions via PHP Attributes.
+ *
+ * @see https://www.php.net/manual/en/language.attributes.php
+ */
+interface RouteAttributesRegistrarInterface
```

Comment:
Are you sure that interface needed?

Reply:
Yes, I've added an interface so that everyone can implement their own registrar routes via attributes.

Reply:
Where will be used this interface?

Reply:
https://github.com/yiisoft/demo/pull/589